### PR TITLE
Feat/clipboard

### PR DESCRIPTION
**Pull request by damien5701**

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -4,15 +4,18 @@
...
+    "clean:android": "rm -rf android/.gradle android/build android/.idea android/*.iml",
     "compile": "tsc -p .",
     "ios": "react-native run-ios",
     "lint": "eslint .",
-    "start": "node node_modules/react-native/local-cli/cli.js start --reset-cache",
+    "start": "node node_modules/react-native/local-cli/cli.js start --reset-cache --port 8082",
     "test": "jest"
   },
   "dependencies": {
+    "@react-native-clipboard/clipboard": "^1.6.0",
```
Comment (imranMnts) [inline example/package.json:15]: already set as dependencies in netwatch, do we need to add it here too?

File: src/Components/Main.tsx [hunk 2]
```diff
@@ -26,48 +19,57 @@ export interface IProps {
...
 
-let reduxList: ReduxAction[] = [];
-let rnList: RNRequest[] = [];
-// let nList: NRequest[] = [];
+const _positionFlatList = 0;
 
 export const Main = (props: IProps) => {
+  const theme = useContext(ThemeContext);
+  const refFlatList = useRef<FlatList<any>>(null);
+  const [flatListLastOffset, setFlatListLastOffset] = useState<number>(0);
```
Comment (imranMnts) [inline src/Components/Main.tsx:32]: it's the fix related to flatlist position? is it fully OK to use in library?

File: src/Components/Main.tsx [hunk 3]
```diff
@@ -86,145 +88,200 @@ export const Main = (props: IProps) => {
...
-  });
+      return false;
+    });
 
   const clearList = () => {
     props.clearAll();
     setRequests([]);
   };
 
+  const formatSharedMessage = (array: Array<ILog>): string => {
```
Comment (imranMnts) [inline src/Components/Main.tsx:125]: should be moved into utils/helpers

File: src/Components/Main.tsx (hunk 2 above)
Comment (damien5701) [reply src/Components/Main.tsx:32]: It works well if you add just one element in the list. However, if there are more calls in one time, the position will be lost. It's only for Android.

File: src/Components/Main.tsx (hunk 3 above)
Comment (damien5701) [reply src/Components/Main.tsx:125]: You are right, can I move this part in my current branche? (feat/share)

File: example/package.json (hunk 1 above)
Comment (damien5701) [reply example/package.json:15]: Removed in feat/share

**Pull request merged**